src/_components/ModelNumber/ModelNumberContainer.jsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import ModelNumberComponent from '../../components/ModelNumber/ModelNumberComponent';
+import { ToastMessage } from '../../_utils/ToastMessage';
+
+import { loadAssetMakes } from '../../_actions/assetMakes.actions';
+import { createModelNumbers } from '../../_actions/modelNumbers.actions';
+import resetToastMessageContent from '../../_actions/resetToastMessage.actions';
+
+class ModelNumberContainer extends React.Component {
+  constructor(props) {
```

Comment:
Use of the constructor is unnecessary if there's no state initialized with props. You can declare state as a class property like so:
```
class ModelNumberContainer extends React.component {
  state = {
    modelNumber: '',
    assetMake: '',
    saveButtonState: false,
    ...
  }

  static getDerivedStateFromProps(nextProps, prevState) {...}
```